config/cmake/CTestCustom.cmake

```diff
@@ -1,19 +1,28 @@
 # CTestCustom.cmake
-set (CTEST_CUSTOM_MAXIMUM_NUMBER_OF_WARNINGS 1500)
+set (CTEST_CUSTOM_MAXIMUM_NUMBER_OF_WARNINGS 3000)
```

Comment:
Ugh. Windows definitely needs a warning reduction pass. Not sure if I'll have time to do that before the release cutoff date, though. I'll try to take a look to see if there's any low-hanging fruit that gets us a big win with little work.